lib/src/widgets/movie_appbar_search_bar.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import '../blocs/i_movies_bloc.dart';
+import '../utils/movie_strings.dart';
+
+class MovieAppBarSearchBar extends StatefulWidget {
```

Comment:
What happen if you want to use MovieAppBarSearchBar elsewhere not necessarily on appBar? I think you can change this name, for MovieSearchBar, because the idea of abstract a widget is that you can use it in more than one place not only on AppBar.

Reply:
You are totally right. Fixed.